tests/test_hear_cleanup.py

```
@@ -166,6 +171,67 @@ def test_cleanup_on_exit(self) -> None:
+			Uses 203.0.113.1 (TEST-NET-3) for RFC 5737 compliance.
+			Port 59095 is chosen as an arbitrary high port number.
+		"""
+		s = None
+		try:
+			# Create a socket connection to an external address
+			s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+			# Connect to a public non-routable IP
+			s.connect(("203.0.113.1", 59095))
+			# Get the IP address of the default interface
+			ip = s.getsockname()[0]
+		except socket.error as _cause:
```

Ruff (0.11.9)

Replace aliased errors with `OSError`

Replace `socket.error` with builtin `OSError`

(UP024)